_dashboards/management/management-index.md

```diff
@@ -20,7 +20,7 @@ Dashboards Management serves as the command center for customizing OpenSearch Da
 
 The following applications are available in Dashboards Management:
 
-- **Index Patterns:** To access OpenSearch data, you need to create an index pattern so that you can select the data you want to use and define the properties of the fields. An index pattern can point to one or more indexes, data streams, or index aliases. 
+- **[Index Patterns]({{site.url}}{{site.baseurl}}/dashboards/management/index-patterns/):** To access OpenSearch data, you need to create an index pattern so that you can select the data you want to use and define the properties of the fields. The Index Pattern tool gives you capabilities to create an index pattern from within the user interface. Index patterns point to one or more indexes, data streams, or index aliases. 
```

Comment:
"The Index Pattern tool" - I tend to refer to this part of the app as "Index Pattern Management". But I think @kgcreative may be the right person to weigh in on how we refer to it.

"so that you can select the data you want to use and define the properties of the fields." - I might say "To access OpenSearch data, you need to create index patterns, which define how data is retrieved and fields are formatted.